eberron.md

```diff
@@ -235,7 +244,7 @@ Increase the range by 10 tiles for every level cast above 1st.
 - **Range:** self
 - **Duration:** 10 minutes, concentration
 
-Form a light layer of bark around your body, creating an extra layer of protection. Make a spellcasting roll against. You gain an extra defense die for every crit.
+Form a light layer of bark around your body, creating an extra layer of protection. Make a spellcasting roll against 2 bad dice. On success you gain 1 defense die, improving a defense to a superior defense for every critical rolled.
```

Comment:
Unbound defense dice is a nightmare for balance. Just look at Jean.

Regardless of who this is cast on, they have at least 2 normal defense dice (3 if you keep this spell adding a defense die), so crits improving dice should always be useful.